.github/workflows/po4a-check.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           if [[ `git status --porcelain translations` ]]; then
             echo "There are uncommited changes to translations:"
             git status --porcelain translations
-            git diff | tee po4a.patch
+            git diff | tee po4a.patch | git --no-pager diff --color=always
```

Comment:
Is this meant to be a separate line? I didn't know `git diff` accepts anything from stdin. Also in this case need to replace tee with > to avoid duplicate diff, right?

Reply:
The purpose is to show a colorized diff and redirect an untainted diff to the patch file. It works as intended, but now that I look at it, the `--no-pager` might be redundant. Yes, `git diff` accepts input from stdin, I learned that recently.

Reply:
I'd like a proof that it does. In my test which I've just done, it doesn't.

Reply:
![image](https://github.com/user-attachments/assets/909830ff-bf87-4670-9d64-e8c4da68ff77)

Reply:
Do this instead:

```sh
sed -i -e 's/errors/beers/' main_en.tex
git diff
git diff > patch
git reset --hard
git diff
cat patch | git diff
```

In your case it "works" not because git diff reads stdin, but because it reads the state of files again, just like the git diff before it

Reply:
Git can read from stdin anyway:
```sh
sed -i -e 's/errors/beers/' main_en.tex
cp main_en.tex main_beer.tex
git reset --hard
cat main_beer.tex | git diff --no-index -- - main_en.tex
```
I'm not arguing which case is which, but it definitely works on github. Are you suggesting any change?